src/main/java/com/Autopark/infrastructure/threads/configurators/ScheduleConfigurator.java

```diff
@@ -0,0 +1,88 @@
+package com.Autopark.infrastructure.threads.configurators;
+
+import com.Autopark.infrastructure.configuratots.ProxyConfigurator;
+import com.Autopark.infrastructure.core.Context;
+import com.Autopark.infrastructure.threads.annotations.Schedule;
+import lombok.SneakyThrows;
+import net.sf.cglib.proxy.Enhancer;
+import net.sf.cglib.proxy.MethodInterceptor;
+import net.sf.cglib.proxy.MethodProxy;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+public class ScheduleConfigurator implements ProxyConfigurator {
+    @Override
+    public <T> T makeProxy(T object, Class<T> implementation, Context context) {
+        for (Method method : implementation.getDeclaredMethods()) {
+            if (method.isAnnotationPresent(Schedule.class)) {
+                checkPublicVoid(method);
+                return (T) Enhancer.create(implementation, (MethodInterceptor) this::invoke);
+            }
+        }
+
+        return object;
+    }
+
+    @SneakyThrows
+    private Object invoke(Object object, Method method, Object[] args, MethodProxy methodProxy) {
+        Schedule scheduleSync = method.getAnnotation(Schedule.class);
+        if (scheduleSync != null) {
+            System.out.println(method);
+            Thread thread = new Thread(() -> this.invoker(object, methodProxy, args, scheduleSync.timeout(), scheduleSync.delta()));
+
+            thread.setDaemon(true);
+            thread.start();
+
+            return null;
+        }
+
+        return methodProxy.invokeSuper(object, args);
+    }
+
```

Comment:
please, do it in the similar style

use here @SneakyThrows also for previous

Reply:
fixed